doc/source/parameters/dem/floating_mesh.rst

```diff
@@ -8,7 +8,7 @@ Floating meshes (solid objects) are finite (limited) auxiliary objects that can
 3. floating wall is stationary while floating mesh may be stationary or moving.
 
 .. note:: 
-    At the moment, solid objects (floating meshes) in Lethe have to be defined using triangular (simplex) meshes. Only triangular 2D meshes of 3D surfaces in the dem_3d solver are presently supported. Quadrilateral 2D meshes of 3D surfaces and 1D mesh of 2D surfaces are not supported at the moment.
+    At the moment, solid objects (floating meshes) in Lethe have to be defined using triangular (simplex) meshes. Only triangular 2D meshes of 3D surfaces in the dem solver are presently supported. Quadrilateral 2D meshes of 3D surfaces and 1D mesh of 2D surfaces are not supported at the moment.
```

Comment:
Is this specific to 3D? Maybe we should specify it here..

Reply:
Fixed